src/VisualStudio/Core/Def/Storage/VisualStudioCloudCacheStorageService.cs

```diff
@@ -14,16 +18,35 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Storage
 {
+    /// <summary>
+    /// The storage service is a process wide singleton.  It will ensure that any workspace that
+    /// wants to read/write data for a particular solution gets the same DB.  This is important,
+    /// we do not partition access to the information about a solution to particular workspaces.
+    /// </summary>
+    [Export(typeof(VisualStudioCloudCacheStorageService)), Shared]
     internal class VisualStudioCloudCacheStorageService : AbstractCloudCachePersistentStorageService
```

Comment:
this is now an exported mef singleton for the entire mef composition (the process).